main.c

```diff
@@ -230,6 +230,9 @@ static int generic_handle_cmd(struct tcmu_device *dev,
 						   ASC_READ_ERROR, NULL);
 		} else
 			return SAM_STAT_GOOD;
+	case WRITE_VERIFY:
+		return store->write_and_verify(dev, tcmulib_cmd, iovec,
+					       iov_cnt, offset);
```

Comment:
For this you mean to call the function you added right?

tcmu_emulate_write_verify()

?